File: src/test/java/org/skefir/data/ImportanceFilterOption.java
package org.skefir.data;

import lombok.Getter;

/**
 * фильтр важности в календаре
 */
public enum ImportanceFilterOption implements OptionFilterable {
    HOLIDAYS("Holidays"),
    LOW("Low"),
    MEDIUM("Medium"),
    HIGH("High");

    @Getter
    final String title;

    ImportanceFilterOption(String title) {
        this.title = title;
    }

    @Override
    public String getAltTitle() {
        return getTitle().toUpperCase();
    }

    @Override
    public String toString() {
        return name() + "-" +  getTitle() + "(" + getAltTitle() + ")";
    }

}

File: src/test/kotlin/org/skefir/data/ImportanceFilterOption.kt
package org.skefir.data

import java.util.*

enum class ImportanceFilterOption(private val title: String) : OptionFilterable {
    HOLIDAYS("Holidays"), LOW("Low"), MEDIUM("Medium"), HIGH("High");

    override fun getTitle(): String {
        return title
    }

    override fun getAltTitle(): String {
        return title.uppercase(Locale.getDefault())
    }

    override fun toString(): String {
        return name + "-" + title + "(" + getAltTitle() + ")"
    }
}